custom/conf/app.example.ini

```diff
@@ -646,6 +646,38 @@ LEVEL = Info
 ;; Host address
 ;ADDR =
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+[audit]
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Enable logging of audit events
+;ENABLED = false
+;; Comma seperated list of appenders to log to (possible values: log, file)
+;APPENDER = 
```

Comment:
This "APPENDER" term is a bit unclear to me, also because it is singular but accepts multiple options. Maybe "DESTINATIONS" would be a better name?

I think we could for example have a future default `database`, so in the most complex variant, this would be `database,log,file` destinations.

Reply:
Java log4j uses `appender`, C# serilog uses `sink`. I have no preference what term we choose.

Reply:
https://github.com/go-gitea/gitea/pull/24257/files#r1316415618 should solve it from documentation perspective. It's never good to describe a word with itself :)